test/GCHP/integration/integrationTestExecute.sh

```diff
@@ -201,6 +201,10 @@ for runDir in *; do
 	    . gchp.env                >> "${log}" 2>&1
 	    . checkRunSettings.sh     >> "${log}" 2>&1
 
+	    # Change time cycle flag in HEMCO_Config.rc from EFYO to CYS,
```

Comment:
Thanks @lizziel.  I can fix that in a later commit.  I thought it would be good to keep the files consistent between GCHP and GCC, even though GCHP uses ExtData.

Reply:
Now fixed in commit 549674d25.